tests/testthat/helper-functions.R

```diff
@@ -24,3 +24,9 @@ skip_on_local <- function() {
   }
   skip("Not on CI")
 }
+
+round_numeric <- function(dt) {
+  cols <- colnames(dt)[purrr::map_lgl(dt, is.numeric)]
+  dt <- dt[, (cols) := lapply(.SD, round, 0), .SDcols = cols]
+  return(dt)
+}
```

Comment:
might be a bit clearer as `round_numerics`. Also, `sapply` could replace `purrr::map_lgl`, don't need to specify the default 0 (though okay with telegraphing intent).